package.json

```diff
@@ -1,34 +1,36 @@
-{
-  "name": "powercord",
-  "version": "2.0.0",
-  "description": "A lightweight @discord client mod focused on simplicity and performance",
-  "license": "SEE LICENSE IN LICENSE",
-  "main": "index.js",
-  "scripts": {
-    "test": "jest",
-    "lint": "eslint --ext js --ext jsx src",
-    "plug": "node injectors/index.js inject --no-exit-codes",
-    "unplug": "node injectors/index.js uninject --no-exit-codes",
-    "inject": "node injectors/index.js inject --no-exit-codes",
-    "uninject": "node injectors/index.js uninject --no-exit-codes"
-  },
-  "repository": "https://github.com/powercord-org/powercord.git",
-  "author": "aetheryx",
-  "bugs": {
-    "url": "https://github.com/powercord-org/powercord/issues"
-  },
-  "dependencies": {
-    "codemirror": "^5.57.0",
-    "less": "^3.12.2",
-    "node-watch": "^0.6.4",
-    "sass": "^1.35.2",
-    "stylus": "^0.54.8",
-    "sucrase": "^3.15.0",
-    "unzip-crx": "^0.2.0"
-  },
-  "devDependencies": {
-    "eslint": "^7.8.1",
-    "eslint-plugin-react": "^7.20.6",
-    "jest": "^26.4.2"
-  }
-}
+{
+  "name": "powercord",
+  "version": "2.0.0",
+  "description": "A lightweight @discord client mod focused on simplicity and performance",
+  "license": "SEE LICENSE IN LICENSE",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint --ext js --ext jsx src",
+    "plug": "node injectors/index.js inject --no-exit-codes",
+    "unplug": "node injectors/index.js uninject --no-exit-codes",
+    "inject": "node injectors/index.js inject --no-exit-codes",
+    "uninject": "node injectors/index.js uninject --no-exit-codes",
+    "update": "node injectors/fkill.js npm run unplug && git pull && npm i && npm run plug"

```

Comment:
The `&&` between the `node injectors/fkill.js` and `npm run unplug` seems to be missing.